nf_core/bump_version.py

```diff
@@ -145,7 +145,7 @@ def bump_nextflow_version(pipeline_obj, new_version):
         pipeline_obj,
         [
             (
-                r"nxf_ver: \[[\'\"]?{}[\'\"]?, ''\]".format(current_version.replace(".", r"\.")),
+                r"nxf_ver: \[[\'\"]?{}[\'\"]?, [\'\"]?[\'\"]?\]".format(current_version.replace(".", r"\.")),
```

Comment:
What is this doing? Allowing either `''` or `""`?

Do you need the `?` characters? I feel like this regex would also match `['foo', ]` which doesn't seem right..

Reply:
Yeah I plead guilty of copying that part of the regexes from other locations where it should replace allow `'` or `"` 
Then let's remove the question marks at the other locations as well I guess? as there should always be brackets I guess

Reply:
removed the question marks now

Reply:
Nice, looks good! Don't think that it mattered much anyway so no need to go hunting elsewhere 👍🏻